Dockerfile

```diff
@@ -0,0 +1,42 @@
+# fog-sw BUILDER
+FROM ros:foxy-ros-base as fog-sw-builder
+
+ARG BUILD_NUMBER
+ARG COMMIT_ID
+ARG GIT_VER
+
+# workaround for ROS GPG Key Expiration Incident
+RUN rm /etc/apt/sources.list.d/ros2-latest.list && \
+    apt-get update && \
+    apt-get install -y curl && \
+    curl http://repo.ros2.org/repos.key | sudo apt-key add - && \
+    echo "deb http://packages.ros.org/ros2/ubuntu focal main" > /etc/apt/sources.list.d/ros2-latest.list && \
+    apt-get update
+
+RUN echo "deb [trusted=yes] https://artifactory.ssrc.fi/artifactory/debian-public-local focal fog-sw" >> /etc/apt/sources.list
```

Comment:
nothing is downloaded from artifactory, so this could be removed too

Reply:
Removed and verified that it can be compiled.